diff --git a/subprojects/gst-plugins-bad/gst-libs/gst/vulkan/gstvkdevice.c b/subprojects/gst-plugins-bad/gst-libs/gst/vulkan/gstvkdevice.c
index afd4c59..5534d90 100644
--- a/subprojects/gst-plugins-bad/gst-libs/gst/vulkan/gstvkdevice.c
+++ b/subprojects/gst-plugins-bad/gst-libs/gst/vulkan/gstvkdevice.c
@@ -247,6 +247,14 @@ static const struct extension optional_extensions[] = {
       VK_MAKE_VERSION (1, 0, 0), NEVER_VK_VERSION),
   OPTIONAL_EXTENSION_VERSION (VK_KHR_SAMPLER_YCBCR_CONVERSION_EXTENSION_NAME,
       VK_MAKE_VERSION (1, 0, 0), NEVER_VK_VERSION),
+  OPTIONAL_EXTENSION_VERSION (VK_KHR_EXTERNAL_MEMORY_FD_EXTENSION_NAME,
+      VK_MAKE_VERSION (1, 0, 0), NEVER_VK_VERSION),
+  OPTIONAL_EXTENSION_VERSION (VK_EXT_EXTERNAL_MEMORY_DMA_BUF_EXTENSION_NAME,
+      VK_MAKE_VERSION (1, 0, 0), NEVER_VK_VERSION),
+  OPTIONAL_EXTENSION_VERSION (VK_EXT_IMAGE_DRM_FORMAT_MODIFIER_EXTENSION_NAME,
+      VK_MAKE_VERSION (1, 0, 0), NEVER_VK_VERSION),
+  OPTIONAL_EXTENSION_VERSION (VK_KHR_EXTERNAL_SEMAPHORE_FD_EXTENSION_NAME,
+      VK_MAKE_VERSION (1, 0, 0), NEVER_VK_VERSION),
 #if defined(VK_KHR_get_physical_device_properties2)
   OPTIONAL_EXTENSION_VERSION
       (VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME,
